lib/controllers/app_controller.dart

```diff
@@ -6,6 +6,13 @@ import 'package:jais/controllers/datas/data_collection_controller.dart';
 import 'package:jais/controllers/url_controller.dart';
 import 'package:jais/firebase_options.dart';
 
+void main() async {
+  WidgetsFlutterBinding.ensureInitialized();
```

Comment:
WidgetsFlutterBinding is already initialized in the main method of main.dart.
It is not, therefore, necessary to reset it.